cluster/examples/kubernetes/ceph/upgrade-from-v1.2-apply.yaml

```diff
@@ -220,4 +220,18 @@ rules:
   resources:
   - csidrivers
   verbs:
-  - create
\ No newline at end of file
+  - create
+---
+# Rook Ceph Operator Config ConfigMap
+# Use this ConfigMap to override Rook-Ceph Operator configurations.
+# NOTE! Precedence will be given to this config if the same Env Var config also exists in the
```

Comment:
Sounds like a scary feature with "NOTE!" :)

Reply:
It's an important note in my mind. Precedence is important.